libs/ui/src/accessible_controllers/mark_scan_controller_illustration.tsx

```diff
@@ -1,42 +1,69 @@
-export type MarkScanControllerButton =
-  | 'up'
-  | 'down'
-  | 'left'
-  | 'right'
-  | 'select'
-  | 'rate-up'
-  | 'rate-down'
-  | 'volume-up'
-  | 'volume-down'
-  | 'help'
-  | 'pause';
+import styled from 'styled-components';
+
+import { Keybinding } from '../keybindings';
+import { MarkScanControllerButton } from './types';
 
 interface MarkScanControllerIllustrationProps {
   highlight?: MarkScanControllerButton;
 }
 
-const HIGHLIGHT_FILL = '#885fce';
-export const MARK_SCAN_CONTROLLER_ILLUSTRATION_HIGHLIGHT_FILL = HIGHLIGHT_FILL;
+const BUTTON_CLASS_NAME = 'markScanControllerIllustration--Button';
+const BUTTON_CLASS_NAME_HIGHLIGHTED = `${BUTTON_CLASS_NAME}--highlighted`;
+const BUTTON_FOREGROUND_CLASS_NAME = `${BUTTON_CLASS_NAME}Foreground`;
```

Comment:
any reason to not delineate with `--` like the other class names?

Reply:
Ah, didn't mean to have the `--` in the `BUTTON_CLASS_NAME` definition - should be `markScanControllerIllustrationButton` there, whereas the `--` suffixes are intended to signify style variations for the base class. Will fix